Flaky scheduler integration tests running against real LSF

**Describe the bug**
Some of the scheduler integration tests are timing out and failing when running against the real LSF cluster. 
```
FAILED integration_tests/scheduler/test_generic_driver.py::test_submit[LsfDriver] - Failed: Timeout >360.0s
FAILED integration_tests/scheduler/test_lsf_driver.py::test_submit_to_named_queue - Failed: Timeout >360.0s
```
(https://github.com/equinor/komodo-releases/actions/runs/8596028934/job/23554954383)

**Expected behaviour**
Tests not failing.


**Environment**
 - OS: RHEL8
 - ERT/Komodo release: ert-onprem-2024.04.rc0-py3.11-rhel8
 - Python version: 3.11
 - Remote/HPC execution involved: Yes, onprem runner.

--- a/ci/testkomodo.sh
+++ b/ci/testkomodo.sh
@@ -67,7 +67,7 @@ start_tests () {
     # Using presence of "bsub" in PATH to detect onprem vs azure
     if which bsub >/dev/null && basetemp=$(mktemp -d -p ~/pytest-tmp); then
         export _ERT_TESTS_ALTERNATIVE_QUEUE=short
-        pytest -v --lsf --basetemp="$basetemp" integration_tests/scheduler
+        pytest --timeout=3600 -v --lsf --basetemp="$basetemp" integration_tests/scheduler
         rm -rf "$basetemp" || true
     fi
     if ! which bsub 2>/dev/null && basetemp=$(mktemp -d -p ~/pytest-tmp); then
@@ -77,7 +77,7 @@ start_tests () {
         else
             export _ERT_TESTS_DEFAULT_QUEUE_NAME=permanent_8
         fi
-        pytest -v --openpbs --basetemp="$basetemp" integration_tests/scheduler
+        pytest --timeout=3600 -v --openpbs --basetemp="$basetemp" integration_tests/scheduler
         rm -rf "$basetemp" || true
     fi
     popd

--- a/tests/integration_tests/scheduler/test_generic_driver.py
+++ b/tests/integration_tests/scheduler/test_generic_driver.py
@@ -79,7 +79,6 @@ async def test_submit_something_that_fails(driver, tmp_path):
     assert finished_called
 
 
-@pytest.mark.timeout(50)
 async def test_kill(driver, tmp_path):
     os.chdir(tmp_path)
     aborted_called = False

